capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/ui/FragmentImplCallback.java

```diff
@@ -54,4 +57,13 @@ void showAlertDialog(@NonNull final String message,
     @NonNull
     NavController findNavController();
 
+    @NonNull
+    default void safeNavigate(@NonNull final NavDirections navDirections) {
+        try {
+            findNavController().navigate(navDirections);
+        } catch (Exception exception) {
+            Logger logger = LoggerFactory.getLogger(FragmentImplCallback.class);
+            logger.error("Navigation exception " + exception.getMessage());
```

Comment:
[Suggestion]: If this logger does not show a UI message shouldn't we show something to the user, why the flow is blocked and why the next step couldn't be reached?

Reply:
Yes, we can face a crash here if fragment already closed.
So I think it's not good to use this function for all navigation cases, just for this risky ones